src/main/kotlin/org/elm/lang/core/psi/PsiElement.kt

```diff
@@ -65,13 +63,6 @@ fun <T : PsiElement> PsiElement.parentOfType(vararg classes: KClass<out T>): T?
     return PsiTreeUtil.getParentOfType(this, *classes.map { it.java }.toTypedArray())
 }
 
-inline fun <reified T : PsiElement> PsiElement.parentsOfType(): Sequence<T> = parentsOfType(T::class.java)
-
-fun <T : PsiElement> PsiElement.parentsOfType(clazz: Class<out T>): Sequence<T> = parents().filterIsInstance(clazz)
-
-fun PsiElement.parents(): Sequence<PsiElement> = generateSequence(this) { it.parent }
```

Comment:
These functions were unused. `parents` is identical to `ancestors`. I can put `parentsOfType` back in if you think it will prevent someone from accidentally using the IntelliJ version that doesn't exist in WebStorm.